resources/automagic_dashboards/table/GoogleAnalyticsTable.yaml

```diff
@@ -1,5 +1,5 @@
-title: Overview of your [[this]] data from Google Analytics
-transient_title: Here''s an overview of your [[this]] data from Google Analytics
+title: Overview of [[this]] data from Google Analytics
```

Comment:
Not that it really matters in the context of this small-ish PR, but aren't we deprecating GA support?

Reply:
I'll ask around. If so, I can delete it. Might be worth a separate cleanup PR to remove any x-ray templates that we no longer use.